frontend/src/mocks/msw/handlers/boards/boardsResponse.js

```diff
@@ -20,4 +21,45 @@ const sampleBoard2 = {
   announcements: [sampleAnnouncement4, sampleAnnouncement5],
   author: 'Krzysztof',
 };
+
+const sampleBoard3 = {
+  id: 2,
+  boardName: 'Opole',
+  announcements: [sampleAnnouncement4, sampleAnnouncement5],
+  author: 'Krzysztof',
+};
+const sampleBoard4 = {
+  id: 4,
+  icon: avatar1,
+  boardName: 'Chrząszczyżewoszyce',
+  announcements: [sampleAnnouncement4, sampleAnnouncement5],
+  author: 'Krzysztof',
+};
+
+const sampleBoard5 = {
+  id: 5,
+  icon: avatar1,
+  boardName: 'Opole',
+  announcements: [sampleAnnouncement4, sampleAnnouncement5],
+  author: 'Krzysztof',
+};
+const sampleBoard6 = {
+  id: 6,
+  icon: avatar1,
+  boardName: 'Wrocław',
+  announcements: [sampleAnnouncement4, sampleAnnouncement5],
+  author: 'Krzysztof',
+};
+const sampleBoard7 = {
+  id: 7,
+  icon: avatar1,
+  boardName: 'Świdnicka',
+  announcements: [sampleAnnouncement4, sampleAnnouncement5],
+  author: 'Krzysztof',
```

Comment:
Jakbyś mogła ponumeruj id kolejno od 0 bo, ja to wykorzystuję później przy mockach i jak będzie tak to pytając np o boarda z id 7 dostanę sampleBoard6 xD